infra/core/compute/ubuntu-jumpbox.bicep

```diff
@@ -46,19 +46,13 @@ param name string
 @description('The name of the linux PC.  By default, this will be automatically constructed by the resource name.')
 param computerLinuxName string?
 
-@description('Username for the Virtual Machine.')
-param adminUsername string
+@description('Username for the Virtual Machine. NOTE: this is not saved anywhere as Entra is used to manage SSH logins once created.')
+param adminUsername string = 'azureadmin'
```

Comment:
Wouldn't it be better to make this a uniqueString by default - if a customer is using "azureadmin" as a uid, then this would conflict.  Using a truly random name is better.

Reply:
good call. initially I did that but there was a script that relied that as the username; that script is now gone, so it works to just have it random